rock_paper_scissor.dart

```diff
@@ -0,0 +1,110 @@
+import 'dart:io';
+import 'dart:math';
+//Гра "Камінь, ножиці, папір": Користувач грає проти комп’ютера, який випадково вибирає один з варіантів. Визначається переможець.
+
+void main() {
+  //Set user name
+  addUserName();
+  //game flow
+  gameFlow();
+}
+
+//Main data
+
+enum GameElements { rock, paper, scissor }
+
+late String userName;
+late GameElements? userChoice;
+late GameElements compChoice;
+
+var userScore = 0;
+var compScore = 0;
+
+//Flow
+void gameFlow() {
+  while (userChooseHandler()) {
+    //Comp pick item
+    getRandomGameElement();
+    winConditions(userChoice, compChoice);
+  }
+  print('Game Over!');
+}
+
+void getRandomGameElement() {
```

Comment:
this function must return number if you call it get. when you will rewrite your logic without global variables it will be must have